airbyte-integrations/bases/source-test-lib/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM openjdk:14.0.2-slim
+
+# Install Docker to launch worker images. Eventually should be replaced with Docker-java.
+# See https://gitter.im/docker-java/docker-java?at=5f3eb87ba8c1780176603f4e for more information on why we are not currently using Docker-java
+RUN apt-get update && apt-get install -y \
+                          apt-transport-https \
+                          ca-certificates \
+                          curl \
+                          gnupg-agent \
+                          software-properties-common
+RUN curl -fsSL https://download.docker.com/linux/debian/gpg | apt-key add -
+RUN add-apt-repository \
+       "deb [arch=amd64] https://download.docker.com/linux/debian \
+       $(lsb_release -cs) \
+       stable"
+RUN apt-get update && apt-get install -y docker-ce-cli jq
+
+ENV APPLICATION source-test-lib
+
+WORKDIR /app
+
+COPY entrypoint.sh .
+COPY build/distributions/${APPLICATION}*.tar ${APPLICATION}.tar
+
+RUN tar xf ${APPLICATION}.tar --strip-components=1
+
+ENTRYPOINT ["/app/entrypoint.sh"]
+
+LABEL io.airbyte.version=0.1.0
+LABEL io.airbyte.name=airbyte/source-test
```

Comment:
piggbacked on the shared code we have for creating docker images for integrations. not entirely sure that was the right thing. assuming it's not, need figure out how to build this image. right now all of our images are either integrations or part of the compose setup, so building this standalone image is new for our build system.

Reply:
rename to `base-source-test` or something that indicates it's a base?

Reply:
lol. it's not really meant to be a base. maybe i'll just move it to its own package.